integration_test/Dialect/Ibis/end_to_end.mlir

```diff
@@ -0,0 +1,103 @@
+// RUN: circt-opt --ibis-containerize --ibis-tunneling --ibis-lower-portrefs \
```

Comment:
This only checks the return code, yes? Perhaps hold back on the integration test and add something more comprehensive later on? Like a cocotb-driven test. Or just filecheck the verilog output to ensure that the correct modules and port lists get created.

Reply:
Will do the latter (check the verilog output). The intention of the end_to_end test here is to just sanity check and have a point of reference for the sequence of passes that needs to be run for a typical program.